Readme.md

```diff
@@ -7,6 +7,7 @@ Xcode 11
 -------
 |🖥 | Computer Model | CPU | RAM | Fresh Build Time | Incremental Build Time | Xcode Version | Date & Commit Hash | New Build System? |
 |-- | -------------- | --- | --- | ---------------- | ---------------------- | ---- | ---- | ----------------- |
+|⌨️ | [Custom PC](https://github.com/ashfurrow/xcode-hardware-performance/pull/108)| AMD Ryzen 9 3950X 4.3GHz(all cores) | 32 GB | 0:21 | 0:03 | 11.4 | 2020-04-16 ([commit](https://github.com/dangthaison91/xcode-hardware-performance/commit/96f9eda856c171fccbb159bc7c4e8ced720abc31) | :heavy_check_mark:|
```

Comment:
This GitHub link should be to the Eidolon repository, the commit that you used.

Reply:
Shouldn't it be a commit of https://github.com/artsy/eidolon? Also, Custom PC is pointing to a different PR: https://github.com/ashfurrow/xcode-hardware-performance/pull/108

Reply:
Yeah, that's a good point. We should have the readme point to the commit _and_ this PR (see other examples). @dangthaison91 I assume you were using the latest commit from `master` at the time? I can update the readme if you give the :+1:

Reply:
Oops, good catch. @ashfurrow Please, go on :D

Reply:
Awesome, thanks! All done 👍